releases/release-1.31/release-notes/maps/pr-125163-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 125163
+releasenote:
+  text: 'DRA: The Dynamic Resource Allocation (DRA) driver''s daemonset must be deployed
```

```suggestion
  text: 'DRA: The Dynamic Resource Allocation (DRA) driver's DaemonSet must be deployed
```
Is this also an **ACTION REQUIRED**?

---

yes it is. Good catch!
```suggestion
  text: 'DRA: ACTION REQUIRED: The Dynamic Resource Allocation (DRA) driver's DaemonSet must be deployed
```